manifests/init.pp

```diff
@@ -49,6 +49,7 @@
   $web_java_opts    = undef,
   $search_java_opts = undef,
   $config           = undef,
+  $use_hiera        = false,
```

Comment:
@fbretel If you want to add the ability to pass a list of plugins, I would suggest to add a `$plugins` parameter instead that takes a list of plugins, default value would be `{}`. On your side you will include this class with this parameter set to `hiera_hash('sonarqube::plugin', {})`